pgctl/cli.py

```diff
@@ -175,13 +182,13 @@ def __call__(self):
     def __change_state(self, opt, expected_state, xing, xed):
         """Changes the state of a supervised service using the svc command"""
         import sys
-        print(xing, self.services, file=sys.stderr)
+        print(xing, self.services_string, file=sys.stderr)
         self.idempotent_supervise()
         with self.pgdir.as_cwd():
             try:
                 while True:  # a poor man's do/while
-                    svc((opt,) + self.services)
-                    status_list = svstat(*self.services)
+                    svc((opt,) + tuple(service.alias for service in self.services))
```

Comment:
"alias" means something else already.
http://pgctl.readthedocs.org/en/latest/user/quickstart.html#aliases

This would be `service.name`.